rules-java/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/GatherIgnoredFileNamesRuleProvider.java

```diff
@@ -33,16 +32,18 @@
 
 
 /**
- * Read and add all the ignore regexes (when a file matches the regex, it will not be scanned by windup) that are present in the windup runtime.
- * @author mbriskar
+ * Read and add all the ignore regexes that are present in the windup runtime.
+ * Files matched by the regex will not be scanned by Windup.
+ * Adds: IgnoredFileRegexModel's to WindupJavaConfigurationModel.
```

Comment:
This is fine, but if you are going to reference a Model by type in a comment, then you should use {@link IgnoredFileRegexModel}, otherwise refactoring type names will break comments. Please update this.